tests/jointcalTestBase.py

```diff
@@ -0,0 +1,311 @@
+# See COPYRIGHT file at the top of the source tree.
```

Comment:
Why is it "Copyright 2016 University of Washington" when the rest of us write "Copyright 2016 AURA/LSST"?  @timj, what's the Answer?